### PR TITLE
department pojo added

### PR DESCRIPTION
**Pull request by rkpatra201**

### CODE REVIEW

File: src/main/java/com/uuv/candidatehiringeventservice/model/Department.java [hunk 1]
```diff
@@ -6,6 +6,9 @@
   private String name;
   private String displayName;
```
Comment (rkpatra201) [inline src/main/java/com/uuv/candidatehiringeventservice/model/Department.java:8]: @saismitamohanty you should add more fields for this as mentioned in the issue. The issue says you should add 4 fields.
https://github.com/umaa-java-sept-2020/candidate-hiring-event-service/issues/3

Comment (swopnalika) [reply src/main/java/com/uuv/candidatehiringeventservice/model/Department.java:8]: @rkpatra201 Getting the error while pushing to branch again !

image


image

Comment (rkpatra201) [reply src/main/java/com/uuv/candidatehiringeventservice/model/Department.java:8]: git push origin master:ticket_name -f

Comment (rkpatra201) [reply src/main/java/com/uuv/candidatehiringeventservice/model/Department.java:8]: @saismitamohanty

Comment (rkpatra201) [reply src/main/java/com/uuv/candidatehiringeventservice/model/Department.java:8]: @swopnalika

**Pull request closed without merging**